cmd/root.go

```diff
@@ -37,8 +40,12 @@ var rootCmd = &cobra.Command{
 	Long: `BitlyBot converts long ugly links to short and friendly ones.
 It does this by leveraging the public Bitly API.`,
 	Run: func(cmd *cobra.Command, args []string) {
+
+		// set bitly api token
+		bitly.SetToken(bitlyToken)
```

Comment:
This will set the token on the default instance of BitlyAPI here: https://github.com/OscarLuu/bitlybot/blob/618023ec4bcc8b91d068446465abb494a46b335e/pkg/bitly/main.go#L14